starters/graphql-serverless-contentful/src/utils/sqs/sendMessage.test.ts

```diff
@@ -0,0 +1,64 @@
+import { sendMessage } from './sendMessage';
+import { getClient } from './client';
+
+jest.mock('./client', () => ({
+	getClient: jest.fn().mockReturnValue({
+		send: jest.fn().mockResolvedValue({}),
+	}),
+}));
+
+jest.mock('./getQueueUrl', () => ({
+	getQueueUrl: jest.fn().mockResolvedValue('queue-url'),
+}));
+
+describe('sendMessage', () => {
+	it('should return an object with success true and the data returned by the send method', async () => {
```

Comment:
Do we want to keep the testing pattern consistent across the board?

In that case we could change this to something like

```
describe('sendMessage', () => {
  describe('when passed a valid message', () => {
    it('returns success', () => {
      // implementation
    })
  })
})
```